object-detector.yaml

```diff
@@ -1,30 +0,0 @@
-apiVersion: v1
-kind: Pod
-metadata:
-  name: object-detector
-  namespace: default
-  labels:
-    app: object-detector
-spec:
-  restartPolicy: Always
-  containers:
```

Comment:
Why did we remove the restart policy?

Reply:
The `Deployment` type has a restart policy by default, so the line in the yaml isn't needed.